lib/db_utils.pm

```diff
@@ -67,15 +67,20 @@ sub influxdb_push_data {
     my ($url, $db, $org, $token, $data, %args) = @_;
     $args{quiet} //= 1;
     $args{proceed_on_failure} //= 0;
+    $args{timeout} //= 90;
     $data = build_influx_query($data);
+    # connectivity pre-check
+    if (script_run('curl --silent --insecure --fail ' . $url . '>/dev/null', timeout => $args{timeout}, die_on_timeout => 0)) {    # curl error
+        record_info("Check", 'NOK DB URL: ' . $url, result => 'fail');
```

Comment:
```suggestion
        record_info("Check", 'Cannot access database ' . $url, result => 'fail');
```
Please no slang. NOK is an obesity clinic where I live.

Reply:
ok: no nok :blush: